lib/mysqlDaoQueryHelper.js

```diff
@@ -185,7 +186,7 @@ class MysqlDaoQueryHelper {
      */
     cleanSpecial(uncleanValue) {
         let cleanValue = null;
-        if (_.includes(['current_timestamp','now()','is null', 'is not null'],_.toLower(uncleanValue))) {
+        if (['current_timestamp','now()','is null', 'is not null'].indexOf(_.toLower(uncleanValue)) !== -1) {
```

Comment:
I was also surprised.  I'm assuming it's because the native implementation has some performance advantages that lodash is unable to take advantage of, because `_.includes()` can accept a wide variety of inputs.